admin_client.go

```diff
@@ -35,6 +35,7 @@ type AdminClient interface {
 	ListSnapshots(t *hrpc.ListSnapshots) ([]*pb.SnapshotDescription, error)
 	RestoreSnapshot(t *hrpc.Snapshot) error
 	ClusterStatus() (*pb.ClusterStatus, error)
+	GetTableNames(t *hrpc.GetTableNames) ([]*pb.TableName, error)
```

Comment:
Could we change it be as "ListTableNames" just to be consistent with java api?

Reply:
Done